### DISCUSSION TITLE
No module named 'numpy'

### DISCUSSION THREAD

**Discussion by npcalyen** [vercel/community#2715] [Help]
### Summary

Hey! I have deployed my Python Flask code using the serverless function, but I need to use a few modules. The most important one is Obspy, but the first error I am facing is "no module named numpy" error on the log page. Is there a way to resolve this issue and install modules? "npm install" or "pip" commands also don't work in the build command section.

### Steps to Reproduce

no module named numpy

**Comment by ccalvocm** [comment]
Greetings @npcalyen 

All working fine under [Anaconda](https://www.anaconda.com/download)

```
conda create -n obspy python=3.8.16
conda activate obspy
conda install obspy

```



![obspy](https://github.com/vercel/community/assets/52244870/22c3a4f6-29f2-4d42-9021-0b2644c6f8fc)

**Reply by npcalyen** [reply]
> Greetings @npcalyen
> 
> All working fine under [Anaconda](https://www.anaconda.com/download)
> 
> ```
> conda create -n obspy python=3.8.16
> conda activate obspy
> conda install obspy
> ```
> 
> ![obspy](https://user-images.githubusercontent.com/52244870/243200933-22c3a4f6-29f2-4d42-9021-0b2644c6f8fc.png)

Hey @ccalvocm 

Please tell me that you didn't give an example from your localhost. 
-_-
